src/metabase/models/humanization.clj

```diff
@@ -82,10 +83,15 @@
   :visibility :settings-manager
   :audit      :raw-value
   :getter     (fn []
-                (let [strategy (setting/get-value-of-type :keyword :humanization-strategy)]
-                  ;; actual advanced method has been excised. Use `:simple` instead if someone had specified
-                  ;; `:advanced`.
-                  (if (= strategy :advanced)
-                    :simple
-                    strategy)))
+                (let [strategy (setting/get-value-of-type :keyword :humanization-strategy)
+                      valid-values (set (keys (methods u.humanization/name->human-readable-name)))
+                      strategy-is-valid (contains? valid-values strategy)
```

Comment:
clojure allows `?` in var names. `valid-strategy?` is probably a better name here